package.json

```diff
@@ -4,7 +4,7 @@
   "description": "TaskHandler",
   "main": "index.js",
   "scripts": {
-    "dev": "parcel src/index.html",
```

Comment:
I was attempting to test running from express. Please remove if this isn't the correct way to go.